tests/thandle_ut/thandle_ut.c

```diff
@@ -488,4 +488,41 @@ TEST_FUNCTION(THANDLE_FREE_with_t_NULL_returns)
     ///cleanup
 }
```

Comment:
Can you actually add a test for:

THANDLE(T)** x;
*x = malloc(sizeof(THANDLE(T)) * 42);

I suspect it should compile now.

Reply:
Wrote a test

---
In reply to: [392014690](https://github.com/Azure/azure-c-util/pull/20#discussion_r392014690) [](ancestors = 392014690)